### PR TITLE
Added trump plugin

### PR DESCRIPTION
**Pull request by jdowner**

### CODE REVIEW

**Comment by jsbronder** [comment]
{code}
Testing trump.py
test_passive (__main__.TrumpTester) ... ERROR

======================================================================
ERROR: test_passive (__main__.TrumpTester)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_trump.py", line 15, in test_passive
    self._proto.privmsg('tester', base.TEST_CHANNEL, 'why?')
  File "/home/jbronder/scm/delbert/delbert/bot.py", line 142, in privmsg
    self._call(f, user, channel, msg, eb=eb)
  File "/home/jbronder/scm/delbert/test/base.py", line 94, in _call
    args[0](*args[1:], **kwds)
  File "/home/jbronder/scm/delbert/plugins/trump.py", line 70, in comment
    _, comment = self.comments[index][1]
ValueError: too many values to unpack
{code}

**Comment by jdowner** [comment]
Sorry, I made the fix but didn't commit it :/

File: plugins/trump.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    def __init__(self, config={}, seed=None):
+        """
+        Make a Trumpian comment
+
+        @param config   - configuration.
+                            verbosity: probability of commenting
+        """
+        super(Trump, self).__init__('trump')
+
+        self.verbosity = config.get('verbosity', 0.1)
```
Comment (jsbronder) [inline plugins/trump.py]: After a week of listening to delbert, mind dropping this default a bit?

**Comment by jsbronder** [comment]
Push me up a squashed commit and I"ll merge.

**Pull request merged**